doc/nrf/gs_programming.rst

```diff
@@ -87,4 +87,84 @@ Complete the following steps to build |NCS| projects with SES after :ref:`instal
    	In a multi-image build, this allows you to debug the source code of your application only.
 
 If you get an error that a tool or command cannot be found, first make sure that the tool is installed.
-If it is installed, verify that its location is correct in the PATH variable or, if applicable, in the SES settings.
\ No newline at end of file
+If it is installed, verify that its location is correct in the PATH variable or, if applicable, in the SES settings.
+
+.. _gs_programming_cmd:
+
+Building on the command line
+****************************
+
+Complete the following steps to build |NCS| projects on the command line after completing the command-line build setup (see the instructions for :ref:`Windows<build_environment_cli_win>`, :ref:`Linux<build_environment_cli_linux>`, or :ref:`macOS<build_environment_cli_mac>`).
+
+1.    Open a terminal window.
+
+#.    Go to the specific sample or application directory.
+      For example, to build the :ref:`at_client_sample` sample, run the following command to navigate to the sample directory:
+
+      .. code-block:: console
+
+         cd nrf/samples/nRF9160/at_client
+
+
+#.    Build the sample or application using the west command.
+      The development board is specified by the parameter *board_name* in the west command as follows:
+
+      .. parsed-literal::
+         :class: highlight
+
+         west build -b *board_name*
+
+      .. note::
+
+	     To build from a directory other than the sample or application directory, run the west build command with an additional parameter *directory_name*,  specifying the sample or application directory.
+
+      See `Board names <Board names_>`_ for more information on the development boards.
+      To reuse an existing build directory for building another sample or application for another board, pass ``-p=auto`` to ``west build``.
+
```

Comment:
Should there be an additional chapter on configuring your application (i.e. running menuconfig) . 
West supports this by calling the command **west build -t menuconfig**?

I would link directly to this chapter in the documentation: https://docs.zephyrproject.org/latest/guides/kconfig/menuconfig.html (which also lists **west -t build menuconfig** as the command to run)

I don't think you can find documentation on configuring, by following build/zephyr...

Reply:
We have a chapter about this here: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_modifying.html#configuring-your-application
Is anything missing in there?

Reply:
It is listed, but quite convoluted _or a tool like menuconfig or guiconfig to update it_

Reply:
Remember that we are advocating for people to use SES. ;)

Reply:
We are linking to https://docs.zephyrproject.org/latest/guides/kconfig/menuconfig.html from our chapter https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/gs_modifying.html#configuring-your-application.
 I think we can add some information here, about configuring application, as follows: 
If you want to configure your application, run the west command:
west build -t menuconfig. 
See :ref:`configure_application` for additional information about configuring an application.